### ISSUE TITLE
training loss become 0

### ISSUE THREAD

**Issue by jiangzhubo** [open-mmlab/mmaction#82]
i remove some video from ucf 101 ,and maybe 20 kinds left , and try do traininig, however the traininig is like that:

wangl@m10:/data/mmaction$ ./tools/dist_train_recognizer.sh configs/ucf101/tsn_rgb_bninception.py 1 --validate
2019-08-21 17:59:38,556 - INFO - Distributed training: True
2019-08-21 17:59:38,789 - WARNING - The model and loaded state dict do not match exactly

unexpected key in source state_dict: fc.weight, fc.bias

2019-08-21 17:59:42,150 - INFO - Start running, host: wangl@m10, work_dir: /data/mmaction/work_dirs/tsn_2d_rgb_bninception_seg_3_f1s1_b32_g8
2019-08-21 17:59:42,150 - INFO - workflow: [('train', 1)], max: 80 epochs
2019-08-21 18:00:01,131 - INFO - Epoch [1][20/64]	lr: 0.00100, eta: 1:20:23, time: 0.946, data_time: 0.670, memory: 4218, loss_cls: 0.5907, loss: 0.5907
2019-08-21 18:00:16,920 - INFO - Epoch [1][40/64]	lr: 0.00100, eta: 1:13:35, time: 0.792, data_time: 0.521, memory: 4218, loss_cls: 0.0000, loss: 0.0000
2019-08-21 18:00:33,505 - INFO - Epoch [1][60/64]	lr: 0.00100, eta: 1:12:10, time: 0.829, data_time: 0.557, memory: 4218, loss_cls: 0.0000, loss: 0.0000

no matter what i try, the loss always become 0 as training, and why trainining data is only 64 videos?

Do you know what going on?
